sui/gateway.js

```diff
@@ -150,11 +151,47 @@ async function callContract(keypair, client, config, chain, args, options) {
 }
 
 async function approveMessages(keypair, client, config, chain, args, options) {
```

Comment:
How do we setup test for this command for [the test workflow](https://github.com/axelarnetwork/axelar-contract-deployments/pull/339)? It seems to require local axelar network with `MultiProver` contract setup.

Reply:
This can't be tested locally since it requires an e2e setup. That's alright, we can cover the other cmds at least